rand_core/src/error.rs

```diff
@@ -71,18 +102,36 @@ impl Error {
     }
 }
 
+impl fmt::Debug for Error {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        #[cfg(feature="std")] {
+            write!(f, "Error {{ inner: {:?} }}", self.inner)
+        }
+        #[cfg(all(feature="getrandom", not(feature="std")))] {
+            getrandom::Error::from(self.code).fmt(f)
+        }
+        #[cfg(not(feature="getrandom"))] {
```

Comment:
You should use `cfg(not(any(feature="getrandom", feature="std")))`. Same for `Display` impl. Maybe use `cfg_if` to be less error-prone?

Reply:
`std` implies `getrandom`